integration-tests/go.mod

```diff
@@ -197,3 +197,5 @@ require (
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 replace github.com/smartcontractkit/chainlink-starknet/relayer => ../../chainlink-starknet/relayer
+
+replace github.com/smartcontractkit/chainlink-starknet/ops => ../ops
```

Comment:
We should remove this from here and use `go.work` for replacing dependencies across this workspace like this. Consider adding and committing the `go.work` with commented out local replace statements.

Reply:
🤔 I understand the reason for not wanting a `replace` in a `go.mod` file
but the alternative is to have each package import the other - and this requires the commit hashes (and for them to exist in the long run), but since we squash commits, it might have to follow a multi-step process? (since squashed/rebased commits I think can get garbage-collected?)
* in the initial PR, point `integration-tests` and `relayer` to a PR commit hash of `ops`
* squash merge PR
* open new PR pointing `integration-tests` and `relayer` to a commit hash in `develop`

Reply:
Yeah, squashing and rebasing becomes a problem with Go workspaces, there is no way around it. But we also can't keep `replace` statements in the public repo, as it's a footgun waiting to go off.

So maybe we just move forward by disabling Github automated squash/rebase-merges? We can still do them manually and `git push -f`, but the final merge should be made by Github after a green CI - meaning the pointers are correct.

@jmank88 any ideas? :)

Reply:
:thinking: